extensions/amp-image-viewer/0.1/amp-image-viewer.js

```diff
@@ -144,7 +143,8 @@ export class AmpImageViewer extends AMP.BaseElement {
   /** @override */
   onLayoutMeasure() {
     if (this.loadPromise_) {
-      this.loadPromise_.then(() => this.measure());
+      this.loadPromise_.then(() => this.resetImageDimensions_())
+          .then(() => this.unregisterPanningGesture_());
```

Comment:
Context: onLayoutMeasure is used as a proxy for when `<amp-image-viewer>` gets resized due to the window resize, device orientation change or a parent resize. If there's a better (or new) way to do this, I'd love to know. We previously used a mess of debouncing listeners on `orientationchange` (https://github.com/ampproject/amphtml/pull/12424) to handle bugs in iOS. 

This addition of `unregisterPanningGesture` fixes a bug. We are supposed to do this whenever the image gets zoomed out, which `resetImageDimensions` will do.